package-lock.json

```diff
@@ -25,6 +25,7 @@
         "clsx": "^2.1.1",
         "isbot": "^4.4.0",
         "lucide-react": "^0.408.0",
+        "pnpm": "^9.6.0",
```

Comment:
uninstall pnpm

Reply:
I have uninstalled in with the commad i "pnpm remove pnpm" command 
 please kindely checkback

Reply:
![Screenshot 2024-07-23 092749](https://github.com/user-attachments/assets/1c0c857c-658d-407c-8f04-42cd3c9b8627)